treeherder/perf/models.py

```diff
@@ -86,3 +86,64 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return "{} {}".format(self.value, self.push_timestamp)
+
+
+@python_2_unicode_compatible
+class PerformanceAlert(models.Model):
+
+    id = models.AutoField(primary_key=True)
+    repository = models.ForeignKey(Repository)
+    prev_result_set_id = models.PositiveIntegerField()
+    result_set_id = models.PositiveIntegerField()
+    series_signature = models.ForeignKey(PerformanceSignature)
+    is_regression = models.BooleanField()
+    amount_pct = models.FloatField()
+    amount_abs = models.FloatField()
+    prev_value = models.FloatField()
+    new_value = models.FloatField()
+    t_value = models.FloatField()
+
+    class Meta:
+        db_table = "performance_alert"
+        unique_together = ('repository', 'prev_result_set_id', 'result_set_id', 'series_signature')
+
+    def __str__(self):
+        return "{} {} {} {}%".format(self.repository, self.result_set_id,
+                                     self.series_signature, self.amount_pct)
+
+
+class PerformanceAlertSummary(models.Model):
+
+    NEW = 0
+    WONTFIX = 1
+    BACKED_OUT = 2
+    INVALID = 3
+    BUGFILED = 4
+    DUPLICATE = 5
+
+    STATUSES = ((NEW, 'New'),
+                (WONTFIX, 'Won\'t fix'),
+                (BACKED_OUT, 'Backed out'),
+                (INVALID, 'Invalid'),
+                (BUGFILED, 'Bug filed'),
+                (DUPLICATE, 'Duplicate'))
+
+    id = models.AutoField(primary_key=True)
+    status = models.IntegerField(choices=STATUSES, default=NEW)
+    repository = models.ForeignKey(Repository)
+    prev_result_set_id = models.PositiveIntegerField()
+    result_set_id = models.PositiveIntegerField()
+    bugzilla_id = models.PositiveIntegerField(null=True)
```

Comment:
ah, my previous comment was for alerts and you have trumped me by doing it on the alert summary!  well done :)  I had to think for a minute here, but there is a small flaw in this- when we have a larger range (a merge, uplift, pgo) and many alerts- we can have multiple status/bugs for the individual alerts, they don't all map to a single root cause.  Not saying we need to continue to show alerts on merges and uplifts, but for pgo, it is important.